pallets/detach/src/uomo_contento.rs

```diff
@@ -0,0 +1,140 @@
+//! Everything in this crate was copied from: https://crates.io/crates/beefy-merkle-tree/10.0.0
```

Comment:
oh thank you, seriously. I was about to write the same comment.

Reply:
A note: always keep in mind that the code you write today will likely be seen/modified/inspected by someone else months/years from now. And it may also happen that you will not be here anymore to help that person. So, meaningful naming is important in a code base, as a knowledge sharing practice.

Reply:
> Please give a meaningful name to this file.

I named it this because I wanted to emphasize that this is only a temporary arrangement (hopefully)